### PR TITLE
[THORN-2116] Adding a link to the Community page

### PR DESCRIPTION
**Pull request by sberyozkin**

### CODE REVIEW

**Comment by sberyozkin** [comment]
See
https://wildfly-swarm.gitbooks.io/wildfly-swarm-users-guide/getting_involved.html

Now there is a link to the Community page with all the relevant information.

**Review by Ladicek — DISMISSED** [review]

**Comment by sberyozkin** [comment]
Thanks for the approval :-), I'll wait for Sheldon to give this and other PRs his writer review during the next few days (in case he'd like to change some phrases or formatting)

**Comment by Ladicek** [comment]
Absolutely!

File: docs/index.adoc [hunk 1]
```diff
@@ -63,3 +63,8 @@ include::modules/ref_the-usage-txt-file.adoc[leveloffset=+2]
 
 include::reference/index.adoc[leveloffset=+2]
 
+# Getting Involved
+
+Community contributions are always welcome !
```
Comment (tradej) [inline docs/index.adoc]:
```suggestion
Community contributions are always welcome.
```

File: docs/index.adoc [hunk 2]
```diff
@@ -63,3 +63,8 @@ include::modules/ref_the-usage-txt-file.adoc[leveloffset=+2]
...
+
+Please visit a https://thorntail.io/community/[Community] page to learn how to get involved.
```
Comment (tradej) [inline docs/index.adoc]:
```suggestion
For more information about how to get involved, see our https://thorntail.io/community/[Community website].
```

**Review by tradej — CHANGES_REQUESTED** [review]
Stylistic changes requested.

**Comment by sberyozkin** [comment]
Thanks Sheldon :-)

**Review by Ladicek — APPROVED** [review]

**Comment by michalszynkiewicz** [comment]
retest this please

**Comment by sberyozkin** [comment]
Merging this now, given that Ladislav has approved and Sheldon's changes have been applied

**Pull request merged**